sos/report/plugins/vectordev.py

```diff
@@ -0,0 +1,30 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, UbuntuPlugin
+
+
+class Vectordev(Plugin):
```

Comment:
What the `dev` stands for? vector.**dev** URL of the project? I would prefer either class name `Vector` or `VectorDev` - until "vectordev" is treated as one English word (totally non-English native speaker here, so I can't judge).

Reply:
I didn't want to call it Vector since it sounds -too- generic, as the word can be applied to a myriad of things.
Fixed on 5bd2d563d586b4fc9d02f2bcca49cb81b7b493cf